source/Backend/BackendInit.js

```diff
@@ -99,6 +113,73 @@ function setDB(dbReturn) {
     db = dbReturn;
 }
 
+//}
+
+/**
+ * Is called to populate the databse with mockData when one doesn't exist
+ * IS NOW DEPRECATED
```

Comment:
We should discuss with the rest of the group or just the team leads if we want to keep this deprecated code as it is removed in the main branch; for your branch it's lines 116 to 182